libraries/debug-certificate-manager/src/CertificateStore.ts

```diff
@@ -81,10 +84,10 @@ export class CertificateStore {
       throw new Error('Unable to determine the current user\'s home directory');
     }
 
-    this._gcbServeDataPath = path.join(this._userProfilePath, '.gcb-serve-data');
-    FileSystem.ensureFolder(this._gcbServeDataPath);
+    this._serveDataPath = path.join(this._userProfilePath, '.rushstack-serve-data');
```

Comment:
This should maintain backwards compatibility with `.gcb-serve-data`.

It'd be cool if it looked in `.gcb-serve-data` and moved the data to the new folder.

Reply:
This was found to not be backwards compatible, and therefore, we decided on generating a new certificate with a 1 character change in the serial number.